Common/src/Common/Service/Data/MessagingSubject.php

```diff
@@ -13,6 +13,8 @@ class MessagingSubject extends AbstractListDataService
 
     public const SORT_ORDER = 'ASC';
 
+    public const ONLY_ACTIVE = TRUE;
```

Comment:
This line is causing the pipeline to fail due to code sniffer violation (needs to be lowercase true)